systems/memory-hierarchy-2/metrics/README.md

```diff
@@ -0,0 +1,116 @@
+Starting point on my machine.
+
+```sh
+$ go test -bench=.
+
+goos: darwin
+goarch: arm64
+pkg: metrics
+BenchmarkMetrics/Average_age-8              1047           1141434 ns/op
+BenchmarkMetrics/Average_payment-8            64          17101507 ns/op
+BenchmarkMetrics/Payment_stddev-8             33          34107580 ns/op
+PASS
+ok      metrics 5.030s
+```
+
+After pulling out the ages into their own array using the "struct of arrays"
+pattern:
+
+```sh
+BenchmarkMetrics/Average_age-8              2370            502892 ns/op
+```
+This is a 2.27x speed-up.
```

Comment:
The way you documented your improvements here is awesome, I'll have to do the same moving forward!

Reply:
Ha, thanks! I didn't want to lose track of what I tried.